ckanext/security/controllers.py

```diff
@@ -36,8 +36,7 @@ def request_reset(self):
             id = request.params.get('user')
 
```

Comment:
Why do we need the `request_reset ` function in `controllers.py` after deleting `ignore_auth`? Can we remove it?

Reply:
The request_reset function was originally in the extension to override CKAN's default logic. The change was to not disclose whether or not an email address exists in the DB. It should always say check your email.

However, it seems that I removed that logic. I'll make another commit to fix this.

Reply:
@injune1123 I put back the logic to not reveal if usernames exist after requesting a password reset.